src/main/java/Duke/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package Duke;
+
+
+import java.time.LocalDate;
+
+/**
+ * Deadline is a type of task with a deadline timing.
+ */
+public class Deadline extends Task {
+
+    protected LocalDate timeobject;
```

Comment:
Could this name be more descriptive?
```suggestion
    protected LocalDate dueDate;
```